parsers/verilog.c

```diff
@@ -221,6 +225,8 @@ static tokenInfo *tagContents = NULL;
  *   FUNCTION DEFINITIONS
  */
 
+static void updateKind (tokenInfo *const token);
```

Comment:
A minor issue.
`updateKind` here is not a function definition. So I would like you to introduce `FUNCTION PROTOTYPES` section.
See the following example taken from c.c:
```C

/*
*   FUNCTION PROTOTYPES
*/
static void createTags (const unsigned int nestLevel, statementInfo *const parent);

/*
*   FUNCTION DEFINITIONS
*/

/*
*   Token management
*/

static void initToken (tokenInfo* const token)
{
	token->type			= TOKEN_NONE;
```